### PR TITLE
Add linux based android SDK locations to gradle

### PR DESCRIPTION
**Pull request by jobe-m**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -73,7 +73,11 @@ fun guessAndroidSdkPath(): String? {
         System.getenv("ANDROID_HOME"),
         "$userHome/AppData/Local/Android/sdk",
         "$userHome/Library/Android/sdk",
-        "$userHome/Android/Sdk"
+        "$userHome/Android/Sdk",
+        "$userHome/AndroidSDK",  // location of sdkmanager on linux
+        "/usr/lib/android-sdk",  // location on debian based linux (sudo apt install android-sdk)
+        "/Library/Android/sdk/", // some other flavor of linux
+        "/Users/$userHome/Library/Android/sdk"
```
Comment (soywiz) [inline build.gradle.kts:80]: This should be either /Users/$userName or $userHome, right? Because userHome is already the absolute path in the filesysten

Comment (jobe-m) [reply build.gradle.kts:80]: ah, right :)

**Review by soywiz — APPROVED** [review]
LGTM!

**Comment by jobe-m** [comment]
Ok, now I guess it should be ok :)

**Review by soywiz — APPROVED** [review]
Looks good to me :) feel free to merge it

**Pull request merged**